test/Api.Test/AdminConsole/Queries/OrganizationUserUserDetailsQueryTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.Models.Data.Organizations.OrganizationUsers;
+using Bit.Core.Repositories;
+using Bit.Core.Utilities;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using Core.AdminConsole.OrganizationFeatures.OrganizationUsers;
+using Core.AdminConsole.OrganizationFeatures.OrganizationUsers.Requests;
+using NSubstitute;
+using Xunit;
+
+namespace Api.Test.AdminConsole.Queries;
+
+[SutProviderCustomize]
+public class OrganizationUserUserDetailsQueryTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task Get_DowngradesCustomUsers(
+        ICollection<OrganizationUserUserDetails> organizationUsers,
+        SutProvider<OrganizationUserUserDetailsQuery> sutProvider,
+        Guid organizationId)
+    {
+        Get_Setup(organizationUsers, sutProvider, organizationId);
+
+        var customUser = organizationUsers.First();
+        customUser.Type = OrganizationUserType.Custom;
+        customUser.Permissions = CoreHelpers.ClassToJsonData(new Permissions
+        {
+            EditAssignedCollections = true,
+            DeleteAssignedCollections = true,
+        });
+
+        var response = await sutProvider.Sut.GetOrganizationUserUserDetails(new OrganizationUserUserDetailsQueryRequest { OrganizationId = organizationId });
+
+        var customUserResponse = response.First(r => r.Id == organizationUsers.First().Id);
+        Assert.Equal(OrganizationUserType.User, customUserResponse.Type);
```

Comment:
Same issue as above - this is now missing key assertions because sut code has been moved back to the controller. Once you've actioned the feedback above, these tests need to be updated so that we keep the same assertions we have today.